test/e2e/tests/settings-search.spec.js

```diff
@@ -0,0 +1,347 @@
+      },
+    ],
+  };
+  const settings = {
+    general: {
+      urlSubstring: 'general',
+      target: 'Primary Currency',
+    },
+    advanced: {
+      urlSubstring: 'advanced',
+      target: 'State Logs',
+    },
+    contacts: {
+      urlSubstring: 'contact-list',
+      target: 'Contacts',
+    },
+    security: {
+      urlSubstring: 'security',
+      target: 'Reveal Secret',
+    },
+    alerts: {
+      urlSubstring: 'alerts',
+      target: 'Browsing a website',
+    },
+    networks: {
+      urlSubstring: 'networks',
+      target: 'Ethereum Mainnet',
+    },
+    experimental: {
+      urlSubstring: 'experimental',
+      target: 'Token Detection',
+    },
+    about: {
+      urlSubstring: 'about-us',
+      target: 'Terms of Use',
+    },
+  };
+  let found;
+  let url;
+
+  it('should find element inside the General tab', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: 'imported-account',
+        ganacheOptions,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        await driver.clickElement('.account-menu__icon');
+        await driver.clickElement({ text: 'Settings', tag: 'div' });
+        await driver.fill('#search-settings', settings.general.target);
+
+        // Check if element was found
+        found = await driver.isElementPresent({ text: 'General', tag: 'span' });
+        assert.equal(found, true, 'Element was not found');
+
+        // Check if element redirects to the correct page
+        await driver.clickElement({ text: 'General', tag: 'span' });
+        url = await driver.getUrl();
+        assert.equal(
+          url.includes(settings.general.urlSubstring),
+          true,
+          'Item does not redirect to correct page',
+        );
+      },
+    );
+  });
+  it('should find element inside the Advanced tab', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: 'imported-account',
+        ganacheOptions,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        await driver.clickElement('.account-menu__icon');
+        await driver.clickElement({ text: 'Settings', tag: 'div' });
+        await driver.fill('#search-settings', settings.advanced.target);
+
+        // Check if element was found
+        found = await driver.isElementPresent({
+          text: 'Advanced',
+          tag: 'span',
+        });
+        assert.equal(found, true, 'Element was not found');
+
+        // Check if element redirects to the correct page
+        await driver.clickElement({ text: 'Advanced', tag: 'span' });
```

Comment:
Similar comment as above, if you use `clickElement` you're making an implicit assertion.

Reply:
fixed in all places